maxmind-db.gemspec

```diff
@@ -20,9 +20,4 @@ Gem::Specification.new do |s|
     'rubygems_mfa_required' => 'true',
   }
   s.required_ruby_version = '>= 2.5.0'
-
-  s.add_development_dependency 'minitest'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'rubocop'
-  s.add_development_dependency 'rubocop-performance'
```

Comment:
Sure, I personally agree with this cop simply because I find the gemspec syntax a bit annoying and I don't see value in dev dependencies being refelected on rubygems.

But I don't really mind either way.

Reply:
Done.